dashboard/db/migrate/20210719221440_create_backpacks.rb

```diff
@@ -0,0 +1,10 @@
+class CreateBackpacks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :backpacks do |t|
+      t.integer :user_id, index: true, unique: true, null: false
+      t.integer :storage_app_id, null: false
```

Comment:
Will we enforce uniqueness of this at model level?

Reply:
good call, I'll add it here. I think it would be weird to have duplicate storage_app_ids across different users (that would mean something went wrong at the storage_app_id generation level), but doesn't hurt to enforce it here too

Reply:
To clarify, I think it's fine to do at model level if you want, which would leave open option for shared backpacks or something one day. Was just confirming we should do it one place or the other.

Reply:
@jmkulwik do you think it's possible we would ever allow duplicate storage_app_ids? If not then I think we should add uniqueness here, otherwise we should enforce on the model for now.

Reply:
apparently the rails-y way to do this is to enforce uniqueness on the model, so I'll stick with that